ev3dev2/led.py

```diff
@@ -346,3 +385,231 @@ def all_off(self):
 
         for led in self.leds.values():
             led.brightness = 0
+
+    def reset(self):
+        """
+        Put all LEDs back to their default color
+        """
+
+        if not self.leds:
+            return
+
+        for group in self.led_groups:
+            self.set_color(group, LED_DEFAULT_COLOR)
+
+    def animate_stop(self):
+        """
+        Signal the current animation thread to exit and wait for it to exit
+        """
+
+        if self.animate_thread_id:
+            self.animate_thread_stop = True
+
+            while self.animate_thread_id:
+                pass
+
+    def animate_police_lights(self, color1, color2, group1='LEFT', group2='RIGHT', sleeptime=0.5, duration=5, block=True):
```

Comment:
Am I correct that on non-EV3 platforms, you must specify groups because these two don't exist?

Reply:
correct.  My gut says these LED animation methods will probably only really be used on EV3 just because it has much more visible LEDs than all of the other platforms.  I think pistorms has two but they are really small.